lib/db/lowlevel.go

```diff
@@ -304,3 +338,11 @@ func (it *iter) execIfNotClosed(fn func() bool) bool {
 	}
 	return fn()
 }
+
+func debugEnvValue(key string, def int) int {
+	v, err := strconv.ParseInt(os.Getenv("STDEBUG_"+key), 10, 63)
+	if err != nil {
+		return def
+	}
+	return int(v)
```

Comment:
Isn't this possible to overflow on 32bit systems? (even if, probably hardly relevant practically)

Reply:
In principle, yeah, in practice the values are not that large.